src/plugins/intel_npu/tools/protopipe/src/parser/config.cpp

```diff
@@ -134,6 +135,23 @@ static std::string toPriority(const std::string& priority) {
 static ScenarioGraph buildGraph(const std::vector<OpDesc>& op_descs,
                                 const std::vector<std::vector<std::string>>& connections);
 
+static void validateNodeChildren(const YAML::Node& node, std::set<std::string> supportedKeys)
```

Comment:
better not to do this `static`, better to put it in anonymous namespace

Reply:
Done